handlers/time.go

```diff
@@ -114,17 +116,46 @@ func (f *Filter) addTimeList(filterID, userAccessToken, collectionID string, req
 		}
 	}
 
+	var selectedMonths []string
 	var options []string
+	startYearStr := req.Form.Get("start-year-grouped")
+	endYearStr := req.Form.Get("end-year-grouped")
+	startYearInt, err := strconv.Atoi(startYearStr)
+	if err != nil {
+		log.Event(ctx, "failed to convert filter start year string to integer", log.ERROR, log.Error(err))
+		return err
+	}
+	endYearInt, err := strconv.Atoi(endYearStr)
+	if err != nil {
+		log.Event(ctx, "failed to convert filter end year string to integer", log.ERROR, log.Error(err))
+		return err
+	}
+
 	for k := range req.Form {
-		if _, err := time.Parse("Jan-06", k); err != nil {
+		if _, err := time.Parse("January", k); err != nil {
```

Comment:
We continue if this is unsuccessful but what will that do? Just trying to understand the impact.